pages/blog/software-development-workflow.mdx

```diff
@@ -7,6 +7,16 @@ image: "software-development-workflow/video.png"
 tags: [product, engineering]
 ---
 
+## Update (December 2021)
+
+The Grouparoo community is continually working to improve the developer experience for Reverse ETL. Here's our guide to [Getting Started with Grouparoo](https://www.grouparoo.com/docs/getting-started) to lead you through installation, configuration, running, and deploying projects. Developers might choose to set up a project using the graphical interface or continue to use our [CLI](https://www.grouparoo.com/docs/cli).
```

Comment:
I think just get rid of the CLI comment and say that we recommend using our UI via `grouparoo config`